Distributing Your Application with Replicated - Helm + Replicated SDK (#271)
replicatedhq/kots-field-labs: instruqt/distributing-with-replicated/05-validating-the-instsall/assignment.md

---
slug: validating-the-instsall
id: vibqdrrlurin
type: challenge

> title: Observing the Custopmer Instance

Reviewer: ```suggestion
title: Observing the Customer Instance
```

teaser: Observing your customer install
notes:
- type: text
  contents: Gaining insight into your customer's instance
tabs:
- title: Vendor Portal
  type: website
  url: https://vendor.replicated.com
  new_window: true
difficulty: basic
timelimit: 600
---

Now that we've installed the application the way your customer
will, let's turn back to your perspective on what's going on

> in their environment. We're going to log back into the Vendor

Reviewer: ```suggestion
in their environment. We're going to look back into the Vendor
```

> Portal in orderr to check the status of Omozan's instance.

Reviewer: ```suggestion
Portal in order to check the status of Omozan's instance.
```

Logging Into the Vendor Portal
==============================

We're going to return to the Replicated Vendor Portal to look
at the details on the current instance. Click on the Vendor
Portal tab to open up a new browser window and access the
portal. You may still be logged in. If not, log in with these
credentials:

Username: `[[ Instruqt-Var key="USERNAME" hostname="shell" ]]`<br/>
Password: `[[ Instruqt-Var key="PASSWORD" hostname="shell" ]]`

Observing the Customer Instance
===============================

Instance Insights is a feature of the Replicated Vendor portal
that provides you detailed insights into the experience your

> customer is having with your applications. It is automatically

Reviewer 2: ```suggestion
customer is having with your application. It is automatically
```

available when you included the Replicated SDK in your application.
The SDK sends telemetry events to the vendor portal to reflect
details about the application, the cluster it's running on, and
current health indicators.

We're going to get to the Instance Insights for the instance we
just installed by navigating from the Vendor Portal landing page.
If you look at the _Stable_ channel, you'll see there is one
active customer indicated.

![Active Customer in Stable Channel](../assets/active-customer-in-channel.png)

If you click on "Details" you'll be brought to a filtered view of

> the Customer's page showing all customers are entitled to releases

Reviewer: ```suggestion
the Customer's page showing all customers that are entitled to releases
```

on the _Stable_ channel. You'll see the customer Omozan there, and

> you'll noticed shows once instance. The specific status of the

Reviewer: ```suggestion
you'll notice it shows once instance. The specific status of the
```
